jerry-ext/include/jerryscript-ext/extfunc.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef JERRYX_EXTFUNC_H
+#define JERRYX_EXTFUNC_H
+
+#include "jerryscript.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif /* __cplusplus */
+
+/*
+ * Handler registration helper
+ */
+
+jerry_value_t jerryx_extfunc_register_global_function (const jerry_char_t *name_p, jerry_external_handler_t handler_p);
+
+/*
+ * Common external function handlers
+ */
+
+jerry_value_t jerryx_extfunc_assert (const jerry_value_t func_obj_val, const jerry_value_t this_p,
```

Comment:
I don't really like this prefix+prefix notation. I think we have enough names, and there will be no clashes, so this can simply be jerryx_assert